kit/metric/client.go

```diff
@@ -0,0 +1,77 @@
+package metric
```

Comment:
Is there any special reason to create a new folder under kit? we have a prometheus package under the root. Or we can make a package inside the pkgr folder?

Reply:
great question, here's the TLDR I gave @goller  in slack about this:

> yah, was going back and forth on that. we definittely could add it to prom. I decided for a new one for 2 reasons.
> 1. metric is dead obvious for someone who looks at the pkg name (“It probably does metrics…“)
> 2. wasn’t sure how tied we are to prom, if we end up using some other metrics engine, its a dependency issue that no one has to worry about except the metric pkg (although dependency change would force updates elsewhere)